src/view_element.cpp

```diff
@@ -1586,7 +1586,7 @@ void View::DrawRestWhole(DeviceContext *dc, int x, int y, int valeur, bool cueSi
     if (y > (int)staff->GetDrawingY()
         || y < staff->GetDrawingY()
                 - (staff->m_drawingLines - 1) * m_doc->GetDrawingDoubleUnit(staff->m_drawingStaffSize))
-        DrawHorizontalLine(dc, x1, x2, y1, m_doc->GetDrawingStaffLineWidth(staff->m_drawingStaffSize));
+        DrawHorizontalLine(dc, x1, x2, y1, m_doc->GetDrawingStaffLineWidth(staff->m_drawingStaffSize) * 1.75);
```

Comment:
It's hardcoded for now, see: View::DrawLedgerLines
It has to be reviewed later on, as mRests should use the same functions as notes.